llvm/include/llvm/TargetParser/X86TargetParser.def

```diff
@@ -229,9 +230,9 @@ X86_FEATURE       (XSAVE,           "xsave")
 X86_FEATURE       (XSAVEC,          "xsavec")
 X86_FEATURE       (XSAVEOPT,        "xsaveopt")
 X86_FEATURE       (XSAVES,          "xsaves")
+X86_FEATURE_COMPAT(AVX512FP16,      "avx512fp16",            39)
```

Comment:
Can we add this feature as key feature to sapphirerapids now?

Reply:
You mean update getIntelProcessorTypeAndSubtype, right? I think we don't need to do after we already updated the model value for sapphirerapids .

Reply:
https://github.com/llvm/llvm-project/blob/main/llvm/lib/TargetParser/X86TargetParser.cpp#L350
I recall we are different with GCC due to the support of AVX512FP16.

Reply:
Sorry, I misunderstood before. c50fb21 updated. With this update, cpu_specific/dispatch can clarify sapphirerapids and cooperlake now.